PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -29,12 +29,14 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <NoWarn>CS2008</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
@dbolkensteyn we are; the pre-processor project (which is the only project that references it) has a NuGet packages.config file.